examples/springboot/README.md

```diff
@@ -320,5 +320,5 @@ create the production environment.
 
 <!-- @finalBuild @testAgainstLatestRelease -->
 ```
-kustomize build $DEMO_HOME  # | kubectl apply -f -
+kustomize build $DEMO_HOME | kubectl apply -f -
```

Comment:
That's not a typo; the `#` is an intentionally placed shell comment delimitter.

Removing it should cause `kustomize-presubmit-master` to fail, since it's supposed to be running this example, and removing the comment now expects both kubectl to be installed and a live cluster to be accepting configuration.

Since it doesn't fail, something must be wrong.  Turns out there are two blank lines causing the test to skip this part.

Please delete the blank line at L106 and L286, the only blank lines immediately preceeding a fenced code block.

When you do the test run by prow will fail unless you leave the comment in place.

I'd be happy to merge a PR that deletes those two blank lines.